src/App.tsx

```diff
@@ -0,0 +1,18 @@
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+import Header from "./headerr/Header";
+import Home from "./pages/Home";
+import Profile from "./pages/Profile";
+
+function App() {
+  return (
+    <Router>
+      <Header />
```

Comment:
![image](https://github.com/milos135/reactNFTWithMetamask/assets/85948457/d5274dcc-7dd4-4f91-9e2c-c13045bbba58)
why do you have 2 headers and why are ther 2 diff fonts?